_includes/header.html

```diff
@@ -2,7 +2,9 @@
   <div class="mdl-layout__header-row">
     <!-- Title -->
     <span class="mdl-layout-title">
-      Apereo Foundation
+      <a href="/">
```

Comment:
Consider using `{{ site.baseurl }}` which will give an appropriate home link even if site is not hosted at the root url.